torch/csrc/api/include/torch/nn/modules/linear.h

```diff
@@ -53,5 +53,51 @@ class TORCH_API LinearImpl : public Cloneable<LinearImpl> {
 /// module storage semantics.
 TORCH_MODULE(Linear);
 
+/// Options for the `Bilinear` module.
+struct TORCH_API BilinearOptions {
+  BilinearOptions(int64_t in1, int64_t in2, int64_t out);
+  /// The number of features in input 1 (columns of the input1 matrix).
+  TORCH_ARG(int64_t, in1);
+  /// The number of features in input 2 (columns of the input2 matrix).
+  TORCH_ARG(int64_t, in2);
+  /// The number of output features to produce (columns of the output matrix).
+  TORCH_ARG(int64_t, out);
+  /// Whether to learn and add a bias after the bilinear transformation.
+  TORCH_ARG(bool, with_bias) = true;
```

Comment:
I think we can change the arguments to be `in1_features`, `in2_features`, `out_features` and `bias`. The argument names of `torch::nn::Linear` currently doesn't match the Python version and we are planning to fix them very soon.